lib/poise_service/service_providers/dummy.rb

```diff
@@ -42,14 +51,17 @@ def action_start
           # Daemonized, set up process environment.
           Dir.chdir(new_resource.directory)
           Chef::Log.debug("[#{new_resource}] Directory changed to #{new_resource.directory}")
+          ENV['HOME'] = Dir.home(new_resource.user)
           new_resource.environment.each do |key, val|
             ENV[key.to_s] = val.to_s
           end
-          Process.uid = new_resource.user
           Chef::Log.debug("[#{new_resource}] Process environment configured")
           IO.write(pid_file, Process.pid)
```

Comment:
Should this come last? I guess it's really just a console log in the parent process, so probably not a big deal.

Otherwise lgtm :+1: